lnrpc/rpc.proto

```diff
@@ -905,6 +915,58 @@ message SendToRouteRequest {
     Route route = 4;
 }
 
+message ChannelAcceptRequest {
+    /// The pubkey of the node that wishes to open an inbound channel.
+    bytes node_pubkey = 1;
+
+    /// The hash of the genesis block that the proposed channel resides in.
+    bytes chain_hash = 2;
+
+    /// The pending channel id.
+    bytes pending_chan_id = 3;
+
+    /// The funding amount in satoshis that initiator wishes to use in the channel.
+    int64 funding_amt = 4;
```

Comment:
any reason that `int64` was chosen instead of `uint64`? none of these values can be negative

Reply:
Yup they should be uint64